eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-devops.yml

```diff
@@ -0,0 +1,99 @@
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20211215.1
```

Comment:
I think maybe we should declare these repositories in an inline checkout, and add a variable to globals.yml with the build tools version ref, unless we plan to revision this repo independently per pipeline?

Reply:
We can share these as needed but we already have automation that will find and update them all at the same time so I'm not super worried about them being duplicated.